examples/scenarios/file.md

```diff
@@ -0,0 +1,44 @@
+# File
+
+Once you have Stanza installed and running from the [quickstart guide](./README.md#quick-start), you can follow these steps to configure a file to send logs via Stanza.
+
+## Prerequisites
+
+On the host that will be sending logs to Stanza, make sure you know the location of the file you wish to gather logs from.
+
+## Configuration
+
+This is an example config file that can be used in the Stanza install directory. It uses a simple [file operator](/docs/operators/file_input.md) to send logs to Google Cloud utilizing a credentials file that can be generated following Google's documentation [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys).
```

Comment:
I think we should give the name / path to the config file. `/opt/observiq/stanza/config.yaml` for linux and `C:\observiq\stanza\config.yaml` for windows.

Reply:
This probably applies to all scenarios but I have not looked that far yet.

Reply:
linked to the quickstart guide where install directories are specified.